cmd/cli/kubectl-kyverno/apply/apply_command.go

```diff
@@ -127,7 +127,7 @@ func Command() *cobra.Command {
 				}
 			}()
 
-			rc, resources, skipInvalidPolicies, pvInfos, err := applyCommandHelper(resourcePaths, userInfoPath, cluster, policyReport, mutateLogPath, variablesString, valuesFile, namespace, policyPaths, stdin, registryAccess)
+			rc, resources, skipInvalidPolicies, pvInfos, err := applyCommandHelper(resourcePaths, userInfoPath, cluster, policyReport, mutateLogPath, variablesString, valuesFile, namespace, policyPaths, stdin, registryAccess, kubeconfig, context)
```

Comment:
As there are so many args to `applyCommandHelper` func , i would suggest as a best practise to define a structure and use as argument here.

Reply:
please have a look. i addressed this.